app/src/main/res/layout/preference_dialog_refresh_interval.xml

```diff
@@ -22,20 +23,30 @@
             android:layout_height="wrap_content"
             android:layout_marginBottom="16dp"
             android:layout_marginTop="8dp"
-            android:textColor="?android:attr/textColorSecondary" />
+            android:textColor="?android:attr/textColorSecondary"
+            tools:text="@string/prefs_message_refresh_interval_min_dialog_message" />
```

Comment:
This on the other hand should be `android:text=""`, right?

Reply:
Absolutely right. Sorry. Could you fix that?